docs/user_manual/source/control_status_registers.rst

```diff
@@ -1786,3 +1786,9 @@ This applies to the following registers:
  - ``dcsr``
  - ``mie``
  - ``mepc``
+ - ``mtvt``
```

Comment:
Can you please add mscratch and jvt to the list here, to the list in docs/user_manual/source/xsecure.rst , and make sure shadow CSRs are instantiated in the RTL as well?

Reply:
Yes, will add those too.